upload/admin/controller/extension/shipping/zasilkovna.php

```diff
@@ -795,6 +797,8 @@ public function adminMenuExtension(&$route, &$data, &$template)
 	 * @throws Exception
 	 */
 	public function orders() {
+		$this->document->addStyle('../catalog/view/theme/zasilkovna/zasilkovna.css');
```

Comment:
Nelíbí se mi míchání stylů pro frontend a backend, navíc pak nejde vložit za využití DIR_APPLICATION.

Reply:
admin CSS mi nejde vytvořit. Asi to souvisí se sekuritou. Našel jsem někoho kdo to řešil vqmod. To se mi moc nechce dělat.

https://www.kvcodes.com/2013/10/customizing-admin-theme-open-cart/

Reply:
Když dáš css do složky admin/view/stylesheet a použiješ něco jako $this->document->addStyle( DIR_APPLICATION . 'view/stylesheet/zasilkovna.css'); tak to nejde?